### ISSUE TITLE
Static Build

### ISSUE THREAD

**Issue by shalikashvili** [RosettaCommons/binder#143]
On xubuntu 20.04, cmake 3.18.3, ninja 1.10.01, Attempts to build per chapter 2 of the documentation failed after the expected several hours generating 217k lines of log messages ending with "_ninja: build stopped: subcommand failed._". 

 I'm now attempting a static build per section 3.4. libclang-static-build, Everything went without error messages, libraries built, but after `make install`, I still have no **binder** executable. Any ideas on what I'm missing?

**Comment by andriish** [comment]
Hi @shalikashvili ,

I'm not using ninja, but it seems some command fails. Maybe some utility needed for this build is missing.
I think even if the logs are large, it would help to see them.

The static build is an experimental feature and worked only with a very specific setup. Particularly this was tested only on CentOS. Nevertheless, would you post the build logs?

Best regards,

Andrii

**Comment by shalikashvili** [comment]
Hi andriish,

I've gzipped the log of my "Build Binder" run and attached it. This is based on chapter 2 (page 6) of "Binder Documentation, Release 0.1" I have used "$HOME/prefix/build" as my build directory to make it easy to copy/paste commands from the documentation.

Rob
[binder-build.log.gz](https://github.com/RosettaCommons/binder/files/5277554/binder-build.log.gz)

**Comment by shalikashvili** [comment]
andriish,

Also, I have attached the CMake output and error log from the "Static Build" attempt per Section 3.4 of the document. Maybe there is something useful there.
[CMakeOutput.log](https://github.com/RosettaCommons/binder/files/5277715/CMakeOutput.log)
[CMakeError.log](https://github.com/RosettaCommons/binder/files/5277717/CMakeError.log)

**Comment by shalikashvili** [comment]
Since my last post, I worked on resolving dependencies. I obtained and installed the "default" package for llvm + clang from  https://llvm.org Then I (re)executed the "Build Binder" step from Section 2.2 (page 6) of the documentation. This reduced the number of items to be compiled from ~3,600 down to ~700, and successfully built a binder executable (et al).

I haven't performed any tests as yet, but it appears that progress has been made. Perhaps some platform dependency (Ubuntu 20.04) was resolved.
